llvm/test/CodeGen/X86/pr38539.ll

```diff
@@ -23,79 +23,84 @@ define void @f() nounwind {
 ; X86-NEXT:    pushl %esi
 ; X86-NEXT:    andl $-16, %esp
 ; X86-NEXT:    subl $176, %esp
-; X86-NEXT:    movzbl {{[0-9]+}}(%esp), %edx
```

Comment:
@RKSimon @arsenm My PR somehow modifies the assembly generated in this test(which I think is not supposed to happen, given that this test has no calls to cltz intrinsic). Any heads up on why this could happen?

Reply:
Some of the paths for divides introduce ctlzs